ACadSharp/Objects/MultiLeaderAnnotContext.cs

```diff
@@ -360,7 +361,7 @@ public override CadObject Clone()
 		{
 			MultiLeaderAnnotContext clone = (MultiLeaderAnnotContext)base.Clone();
 
-			foreach (var leaderRoot in LeaderRoots)
+			foreach (var leaderRoot in LeaderRoots.ToList())
```

Comment:
Is there a particular case to add the ToList()?

Reply:
see above.